helm/fluent-logshipping-app/templates/configmap.yaml

```diff
@@ -210,6 +210,10 @@ data:
         Time_Key time
         Time_Format %d/%b/%Y:%H:%M:%S %z
 
+    {{- if .Values.extraParsers }}
+    {{- .Values.extraParsers | nindent 4 }}
+    {{- end }}
```

Comment:
I would rather make extraParsers an array and loop through it. So we would then add the `[PARSER]` section ourselves here which ensure the provided configuration is used as a parser and not something else, it also provide a more structured way to define multiple parsers.